vividus-bdd-engine/src/main/java/org/vividus/bdd/output/ManualStepConverter.java

```diff
@@ -98,6 +95,18 @@ private ManualStepConverter()
         return manualSteps;
     }
 
+    public static boolean startsWithManualKeyword(String value)
+    {
+        return value.startsWith(STEP_SIGN);
+    }
+
+    public static List<String> cutManualIdentifier(List<Step> steps)
+    {
+        return steps.stream().map(Step::getValue)
+                             .map(v -> RegExUtils.replaceAll(v, "!--\\s*-?\\s*", StringUtils.EMPTY))
```

Comment:
String org.apache.commons.lang3.RegExUtils.replaceAll(String text, Pattern regex, String replacement), to avoid pattern compilation